DQM/SiStripMonitorClient/python/pset4GenericTriggerEventFlag_cfi.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+genericTriggerEventFlag4HLTdb = cms.PSet(
+   andOr         = cms.bool( False ),
+   dbLabel       = cms.string("SiStripDQMTrigger"), # ("TrackerDQMTrigger"),
+   andOrHlt      = cms.bool(True), # True:=OR; False:=AND
+   hltInputTag   = cms.InputTag( "TriggerResults::HLT" ),
+   hltPaths      = cms.vstring(""), #cms.vstring("HLT_ZeroBias_v*","HLT_HIZeroBias_v*")
+   hltDBKey      = cms.string("SiStrip_HLT"),
+   errorReplyHlt = cms.bool( False ),
+   verbosityLevel = cms.uint32(1)
```

Comment:
please change to 0 to suppress the warnings mentioned in 
https://github.com/cms-sw/cmssw/pull/14659#issuecomment-223442851
unless the warnings are very temporary and a new GT with the keys is coming soon, we don't need to clutter the logfiles